project/PackagingTypePlugin.scala

```diff
@@ -0,0 +1,8 @@
+import sbt._
+
+object PackagingTypePlugin extends AutoPlugin {
```

Comment:
Taken from here I see :)

https://github.com/sbt/sbt/issues/3618#issuecomment-424924293

Might be nice to include the above github link as a comment to help readers of the code.

This workaround worked for me but I had to `sbt clean` first -- probably why the circle build failed, since it isn't cleaning right now as written.